backend/Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM python:3
 ADD . /api
+COPY ./ddueruemweb/.env.production /api/ddueruemweb/.env
 WORKDIR /api
 
 # This is for the performant prod server
 RUN pip install uwsgi
 RUN pip install -r requirements.txt
 ENV PORT=8000
 EXPOSE 8000
+EXPOSE 1025
```

Comment:
Are we still using this port ? I think this was only relevant for starting the "email backend" but since we switched to mailtrap.io, i think we can remove this link

Reply:
👍🏻